database/src/main/resources/db/migration/V116__klagevedtak.sql

```diff
@@ -0,0 +1,8 @@
+create table if not exists klagevedtak
+(
+    id uuid primary key,
+    opprettet timestamptz not null,
+    type text not null,
+    metadata jsonb not null
+);
+create unique index klagevedtak_metadata_hendelseId_idx on klagevedtak ((metadata->>'hendelseId'));
```

Comment:
Sjekk docsen ang. indexering av jsonb i 8.14.4: https://www.postgresql.org/docs/9.4/datatype-json.html
Lurer på om man skal ha sån GIN index?

Reply:
Virker ikke som det støttes for unique indices (med expressions). Fant ingen eksempler på det heller. Jeg sliter og litt med å legge til constraint på kun den unique indexen. Vurderer og bytte det ut med read-conditional write transaksjon.